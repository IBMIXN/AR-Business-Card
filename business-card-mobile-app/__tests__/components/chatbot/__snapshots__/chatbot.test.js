import { NavigationContainer } from "@react-navigation/native";
import { createStackNavigator } from "@react-navigation/stack";
import { fireEvent, render, act } from "@testing-library/react-native";
import * as React from "react";
import renderer from "react-test-renderer";
import ChatBot from "../../../../components/chatbot/chatbot";

// jest.mock("expo-av", () => {
//   return {
//     get: jest.fn().mockImplementation(() => Promise.resolve(true)),
//   };
// });

const Stack = createStackNavigator();

describe("<Chatbot (component) />", () => {
  //   SNAPSHOT
  test("renders correctly", async () => {
    const component = <ChatBot />;

    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
