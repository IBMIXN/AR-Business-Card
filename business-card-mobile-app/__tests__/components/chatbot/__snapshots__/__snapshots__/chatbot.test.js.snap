// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChatbotScreen /> renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "black",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "overflow": "hidden",
      }
    }
  >
    <RCTScrollView
      autoScrollDisabled={false}
      contentContainerStyle={
        Object {
          "paddingBottom": "35%",
        }
      }
      data={Array []}
      disableVirtualization={false}
      getItem={[Function]}
      getItemCount={[Function]}
      horizontal={false}
      initialNumToRender={10}
      keyExtractor={[Function]}
      maxToRenderPerBatch={10}
      onContentSizeChange={[Function]}
      onEndReachedThreshold={2}
      onLayout={[Function]}
      onMomentumScrollEnd={[Function]}
      onScroll={[Function]}
      onScrollBeginDrag={[Function]}
      onScrollEndDrag={[Function]}
      removeClippedSubviews={false}
      renderItem={[Function]}
      scrollEventThrottle={50}
      stickyHeaderIndices={Array []}
      updateCellsBatchingPeriod={50}
      viewabilityConfigCallbackPairs={Array []}
      windowSize={21}
    >
      <View />
    </RCTScrollView>
  </View>
  <View
    style={
      Object {
        "bottom": 30,
        "left": 10,
        "position": "absolute",
        "right": 10,
        "zIndex": 2,
      }
    }
  >
    <View
      style={
        Object {
          "alignSelf": "center",
          "justifyContent": "center",
          "padding": 3,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
        testID="mic-button"
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "white",
              "borderRadius": 50,
              "padding": 20,
            }
          }
        >
          <Text />
        </View>
      </View>
    </View>
  </View>
  <Image
    source={1}
    style={
      Object {
        "bottom": 5,
        "height": 100,
        "left": 15,
        "position": "absolute",
        "width": 100,
        "zIndex": 1,
      }
    }
  />
</View>
`;
